### PR TITLE
core: tools: Update filebrowser to v2.24.2

### PR DESCRIPTION
**Pull request by joaoantoniocardoso**
It might help #1951 and #2048

### CODE REVIEW

**Comment by joaoantoniocardoso** [comment]
Confirmed: 
- closes #1951 
- closes #2048

File: core/Dockerfile [hunk 1]
```diff
@@ -67,7 +67,7 @@ RUN RCFILE_PATH="/etc/blueosrc" \
     && echo "export GIT_DESCRIBE_TAGS=$GIT_DESCRIBE_TAGS" >> $RCFILE_PATH
 
 # create some Symlinks to make life easier
-RUN mkdir /shortcuts
+RUN mkdir -p /shortcuts
```
Comment (Williangalvani) [inline core/Dockerfile:70]: has this ever happened? that is weird.
not a bad patch, I'm just wondering how it could happen.

**Review by Williangalvani — APPROVED** [review]
tested, NICE

File: core/Dockerfile (hunk 1 above)
Comment (joaoantoniocardoso) [reply core/Dockerfile:70]: Yeah, I wonder, initially I thought it was some weird cache from GitHub, but it was also happening on my computer without cache :eyes:

**Pull request merged**